pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/mercury/MercuryOrderDto.java

```diff
@@ -99,7 +99,7 @@ public Builder withCreatedByUserId(String createdByUserId) {
             return this;
         }
 
-        public Builder withTissueId(int tissueId) {
+        public Builder withTissueId(Integer tissueId) {
```

Comment:
Let the null through so caller checks on nullity will work properly.

Reply:
Looks like `MercuryOrderDto` is heavily used.
Do we have a sense that no code is relying on the `0` value?

Reply:
None that I could find.  Although I did find other code that is expecting getTissueId() to be null sometimes, so those paths might be triggered.

Reply:
Although if the objects are created with getInt(), the behavior will remain the same.